.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,123 @@
+name: Publish Release
+
+on:
+  workflow_call:
+    secrets:
+      NPM_TOKEN:
+        required: true
+      SLACK_WEBHOOK_URL:
+        required: true
+      PUBLISH_DOCS_TOKEN:
```

Comment:
We can drop the `PUBLISH_DOCS_TOKEN` secret for now, since it's tied to the documentation-related jobs, and we don't need those right now.

Reply:
Dropped this input parameter form ```publish-release.yaml``` as well se from ```main.yaml``` file that uses this parameter